cypress/e2e/userIsAbleToUseGeofence.cy.js

```diff
@@ -9,7 +9,7 @@ describe("Geofence", () => {
 	beforeEach(() => {
 		cy.visitDomain(`${Cypress.env("WEB_DOMAIN")}/demo`);
 		cy.get('[data-testid="hamburger-menu"]').click();
-		cy.get('[data-testid="connect-aws-account-button"]').click();
+		cy.get('[data-testid="connect-aws-account-button"]').first().click();
```

Comment:
Yes, since the updated data-testid hasn't yet been deployed to the production, that's why first() is being used to take care of that, I'll remove this in the Next PR raised after the data-testid has gone to prod.